xwiki-platform-core/pom.xml

```diff
@@ -229,6 +229,7 @@
     <module>xwiki-platform-release</module>
     <module>xwiki-platform-rendering</module>
     <module>xwiki-platform-repository</module>
+    <module>xwiki-platform-requiredrights</module>
```

Comment:
Sorry if it has been discussed in the past, but feels a bit weird that it has its own root module: I'd have expected either to be in the security-authorization module, or if it really needs its own module, to be a submodule of security.

Reply:
Is has not been discussed. I've placed it here but I'm not against something like `xwiki-platform-security-requiredrights`